server/command.go

```diff
@@ -330,26 +330,18 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 }
 
 func (p *Plugin) runRemoveCommand(args []string, extra *model.CommandArgs) (bool, error) {
-	message := strings.Join(args, " ")
-	listID := MyListKey
-	issueList, err := p.listManager.GetIssueList(extra.UserId, listID)
-	if err != nil {
-		return false, err
-	}
-
-	var issueID string
-	for _, issue := range issueList {
-		if issue.Message == message {
-			issueID = issue.ID
-		}
+	if args[0] != "--todo" {
```

Comment:
We should make sure `len(args) == 2` before accessing `args`

Reply:
Done.